File: src/main/java/net/orandja/vw/accessor/RecipeManagerAccessor.java
package net.orandja.vw.accessor;

import net.minecraft.recipe.Recipe;
import net.minecraft.recipe.RecipeManager;
import net.minecraft.recipe.RecipeType;
import net.minecraft.util.Identifier;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Accessor;

import java.util.Map;

@Mixin(RecipeManager.class)
public interface RecipeManagerAccessor {
    @Accessor("recipes")
    void setRecipes(Map<RecipeType<?>, Map<Identifier, Recipe<?>>> recipes);

    @Accessor
    Map<RecipeType<?>, Map<Identifier, Recipe<?>>> getRecipes();
}

File: src/main/java_old/accessors/RecipeManagerAccessor.kt
@file:Mixin(RecipeManager::class)
@file:JvmName("RecipeManagerAccessor")
package net.orandja.vw.accessors

import com.google.gson.Gson
import net.minecraft.recipe.Recipe
import net.minecraft.recipe.RecipeManager
import net.minecraft.recipe.RecipeType
import net.minecraft.resource.JsonDataLoader
import net.minecraft.util.Identifier
import org.spongepowered.asm.mixin.Mixin
import org.spongepowered.asm.mixin.gen.Accessor

@Mixin(RecipeManager::class)
interface RecipeManagerAccessor {
    @Accessor("recipes")
    fun setRecipes(recipes: Map<RecipeType<*>, Map<Identifier, Recipe<*>>>)

    @Accessor
    fun getRecipes(): Map<RecipeType<*>, Map<Identifier, Recipe<*>>>
}